src/cli/app-instance.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * Worker thread that runs an instance of a mocked local fulfillment app.
+ */
+import {parentPort, workerData} from 'worker_threads';
```

Comment:
curious why you decided to run this in a worker?

Reply:
I discussed with @Fleker doing the CLI synchronously vs using worker threads.  I think doing the intent fulfillment in a worker thread keeps things modular, which is good for adding features down the road.  For example, I could potentially swap out the command processor with a chrome UI.  I also figured it was good practice to keep the UI/View split up from the processing.